src/utils/navigation.ts

```diff
@@ -42,3 +43,26 @@ export const redirectToSavedUrl = (
  * @returns {string} sign in path
  */
 export const buildSignInPath = ({ host }: { host: string }) => `${host}/signin`;
+/**
+ * @param  {string} host target host
+ * @param  {string} itemId id of the item
+ * @param  {boolean} chatOpen whether to have the chat open
+ * @returns {string} lint to item with chat open
+ */
+export const buildItemLinkForBuilder = ({
+  host,
+  itemId,
+  chatOpen,
+}: {
+  host: string;
+  itemId: string;
+  chatOpen?: boolean;
+}) => {
+  const baseUrl = new URL(`${host}/items/${itemId}`);
+  const queryString = new URLSearchParams({
+    ...(chatOpen && { chat: ChatStatus.OPEN }),
+  });
+  // add params to url
+  baseUrl.search = queryString.toString();
+  return baseUrl.toString();
```

Comment:
Not sure why you did it like this aha sorry for not being precise. I was more thinking about qs https://www.npmjs.com/package/qs
So you would have something approx like this:
```
`${host}/items/${itemId}${qs.stringify({chat: chatOpen}, {addPrefix:true})}`
```
which handles the `?` for you.

Reply:
Okay i will use that, i thought that using the `URL` and `URLSearchParam` APIs was better, but this is shorter so :)